pkgs/tools/misc/zellij/default.nix

```
@@ -43,6 +45,9 @@ rustPlatform.buildRustPackage rec {
   '';
 
   postInstall = ''
+    mandown $src/docs/MANPAGE.md > ./zellij.1
+    installManPage ./zellij.1
```

`$src` is unnecessary.
```suggestion
    mandown docs/MANPAGE.md > zellij.1
    installManPage zellij.1
```

---

Thanks! I wasn't sure abut that.